vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java

```diff
@@ -89,4 +96,11 @@ private static TestExecutionItemStatus calculateStatus(AbstractStepsContainer st
                      .allMatch(step -> "successful".equals(step.getOutcome())) ? TestExecutionItemStatus.PASS
                              : TestExecutionItemStatus.FAIL;
     }
+
+    private static String asOffsetDateTime(long nanos)
+    {
+        long millis = TimeUnit.MICROSECONDS.convert(nanos, TimeUnit.NANOSECONDS);
+        Instant instant = Instant.ofEpochSecond(millis);
+        return OffsetDateTime.ofInstant(instant, ZoneId.systemDefault()).toString();
```

Comment:
how is it displayed in XRay? is user's TimeZone displayed somewhere? or is it converted to the server timezone?

Reply:
Jira/Xray parses the provided date and displays it using own format